api/service.py

```
@@ -710,276 +834,91 @@ async def get_anomaly_history(
-        ts_data = TimeSeriesData(
-            timestamp=datetime.now(),
-            cpu_usage=telemetry.cpu_usage or 0.0,
-            memory_usage=telemetry.memory_usage or 0.0,
-            network_latency=telemetry.network_latency or 0.0,
-            disk_io=telemetry.disk_io or 0.0,
-            error_rate=telemetry.error_rate or 0.0,
-            response_time=telemetry.response_time or 0.0,
-            active_connections=telemetry.active_connections or 0,
-            failure_occurred=False  # We're predicting, not reporting actual failure
+@app.get("/api/v1/auth/apikeys", response_model=List[APIKeyResponse])
+async def list_api_keys(current_user: User = Depends(get_current_user)):
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)